### ISSUE TITLE
During account setup, "reveal" password doesn't work in horizontal mode

### ISSUE THREAD

**Issue by znoteer** [juha-h/baresip-studio#156]
baresip+ 17.0.0 on calyxOS 2.4.0 (AOSP 11) on a Pixel3, and 
baresip 30.0.0 on android 6.0 on an LG G3

If one enters a password in the account setup screen with the phone in portrait mode, one can choose to reveal or hide the password while it is being entered.  If holding the phone in landscape mode, there is no icon for choosing between reveal or hide.

Also, if one starts in vertical mode with password visible but then switches to horizontal mode, the password becomes hidden.  Turning the phone back to portrait mode reveals the password again.

In short, it is not possible to see the password during acct setup while typing in horizontal mode. (Big fingers don't like typing on the tiny keyboards of vertical mode)

**Comment by juha-h** [comment]
Thanks for reporting.  This should now be fixed by commit https://github.com/juha-h/baresip-studio/commit/1770af11daf37049db5a4a4fa72c60f13198cfe9.

When entering Backup/Restore passwords, there is not enough vertical room (at least in my devices) for the toggle nor the password to show up in horizontal mode  if the same solution is used.   So I have not made any changes there.

**Comment by juha-h** [comment]
Made Backup/Restore password dialogs scrollable, which now allows togging of password visibility.

New releases 30.1.0 and 17.1.0 has these fixes included.  I'll close this now.  Reopen if needed once you have installed the new releases.
